### PR TITLE
Cleanup Dockerfiles, Makefiles etc of microfrontends

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Remove Dockerfiles etc. from single microfrontends

**Related issue(s)**

### CODE REVIEW

File: resources/kustomization.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - kubernetes-dashboard-admin.yaml
```
Comment (valentinvieriu) [inline resources/kustomization.yaml]: Do we need this kubernetes-dashboar?

**Review by valentinvieriu — CHANGES_REQUESTED** [review]

File: resources/kustomization.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
```
Comment (valentinvieriu) [inline resources/kustomization.yaml:3]: This file can be removed also

**Review by valentinvieriu — CHANGES_REQUESTED** [review]

File: Dockerfile-all [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# this is a Dockerfile for single deployment app - both backend and frontends
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply Dockerfile-all]: Oh, I've renamed it on my previous PR. On it.

File: resources/kustomization.yaml (hunk 1 above)
Comment (Wawrzyn321) [reply resources/kustomization.yaml]: :pick:

**Review by valentinvieriu — APPROVED** [review]

**Pull request merged**